pybikes/bysykkel.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import json
+
+from .base import BikeShareSystem, BikeShareStation
+from . import utils
+
+
+class BySykkel(BikeShareSystem):
+
+    meta = {
+        'system': 'BySykkel',
+        'company': ['Urban Infrastructure Partner']
+    }
+
+    def __init__(self, tag, meta, feed_url, feed_details_url, feed_token):
+        super(BySykkel, self).__init__(tag, meta)
+        self.feed_url = feed_url
+        self.feed_details_url = feed_details_url
+        self.feed_token = feed_token
```

Comment:
If a system is using a key, it should set its class attribute as `authed = True`. See [cyclocity.py](https://github.com/eskerda/pybikes/blob/master/pybikes/cyclocity.py) as an example